migrate.py

```diff
@@ -37,42 +38,44 @@
 
 """
 
+reload(sys)
```

Comment:
I'm not super fond of this:

http://stackoverflow.com/questions/3828723/why-should-we-not-use-sys-setdefaultencodingutf-8-in-a-py-script